packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -469,6 +480,15 @@
             padding-bottom: 0.4rem;
         }
     }
+    &__limit-order-info-mobile {
```

Comment:
__limit-order-info-mobile can be nested inside &__limit-order-info {, please check all classnames that you added, we can nest like this for example:
```
&__limit-order-info {
    // some styles for __limit-order-info

    &-mobile {
        // some styles for __limit-order-info-mobile
    }
}
```

Reply:
I rewrote style, make them nestable